XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -414,21 +414,31 @@
               "Title": "ListView - API",
               "Uri": "https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.listview"
             },
+            {
+              "Title": "Guidelines",
+              "Uri": "https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/listview-and-gridview"
+            },
+            {
+              "Title": "Drag and Drop - Full Sample",
+              "Uri": "https://github.com/microsoft/Windows-universal-samples/tree/master/Samples/XamlDragAndDrop"
+            },
+            {
+              "Title": "CollectionViewSource - API",
+              "Uri": "https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Data.CollectionViewSource#see-also"
+            },
             {
               "Title": "Inverted Lists",
               "Uri": "https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/inverted-lists"
             },
             {
               "Title": "Inverted Lists - Full Sample",
               "Uri": "https://github.com/Microsoft/Windows-universal-samples/tree/master/Samples/XamlBottomUpList"
-            },
```

Comment:
Why is this removed?

Reply:
Not sure why the preview is showing this - I added some more links, but the JSON structure is still intact.